templates/zerver/help/add-a-bot-or-integration.md

```diff
@@ -0,0 +1,41 @@
+# Add a Bot or Integration
+
+Bots and integrations are features that are accesible to all members of an organization.
+
+## To Add a Bot
+
+Bots can be created from the settings page:
+
+{!go-to-the.md!} [Your Bots](/#settings/your-bots) {!settings.md!}
+
+2. On this page, under the **Add a Bot** window, you can enter a name, an email, and
+optionally an avatar for your bot. Then click the **Create Bot** button to create your bot.
+
+    ![Bots page](/static/images/help/bots_settings_page.png)
+
+3. An entry such as the following should appear above the **Add a New Bot** view, confirming
+the creation of your bot.
+
+    ![Example bot](/static/images/help/bot_example.png)
+
+4. You have succesfully created your bot. You now have access to your bot's API key and API
+configuration file (`.zuliprc`) which you will need for integrations that you would like
+to use with this bot. You can click on the green download icon to download your `.zuliprc` file.
+
+    ![Download .zuliprc](/static/images/help/download_zuliprc.png)
+
+## To Add an Integration
+
+There is a full list of integrations and the directions to add them on the Integrations page.
+You can access this page by following the given instructions.
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner
+of the right sidebar.
+
+2. Next, select **Integrations** from the dropdown that appears.
```

Comment:
But if there isn't a macro for integrations, I need to be providing instructions to get to the instructions page. That's the whole point.